tests/suites/sanity/sanity_test.go

```diff
@@ -90,6 +96,55 @@ var _ = Describe(TestName, func() {
 		assertNumberOfCassandraNodes(NodeCount)
 	})
 
+	It("provides metrics to prometheus", func() {
+		podTpl := v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "curl-pod",
+				Namespace: "default",
+			},
+			Spec: v1.PodSpec{
+				Containers: []v1.Container{
+					{
+						Name:    "curl",
+						Image:   "curlimages/curl",
+						Command: []string{"sleep", "300"},
+					},
+				},
+			},
+		}
+
+		pod, err := kubernetes.NewPod(Client, podTpl)
+		Expect(err).To(BeNil())
+
+		for {
```

Comment:
IMO, this is okay for this test. But if for any reason the pod doesn't get into `PodRunning` phase, we would loop until we run into a timeout. I'd rather have a loop that can timeout here, like in https://github.com/kudobuilder/test-tools/blob/master/pkg/kudo/instance.go#L65. Though this looks like a feature for `test-tools`, you could also create an issue there ;)